regexes/oss.yml

```diff
@@ -160,6 +160,13 @@
   name: 'MTK / Nucleus'
   version: '$1'
 
+##########
+# MRE / MAUI Runtime Environment
+##########
+- regex: 'MRE/(\d+)\.(\d+).*;MAUI'
+  name: 'MRE'
+  version: '$1.$2'
```

Comment:
`MRE` not sure about the correct name, MRE is more of an SDK (created in Mediatek)

Reply:
Not sure here either. Tried to find some useful information on that, but couldn't :shrug: Other parsing libraries are detecting it as `MRE`, so might be fine.